demo/components/FileInput.tsx

```diff
@@ -0,0 +1,56 @@
+import * as yaml from 'js-yaml';
+import * as React from 'react';
+import { RefObject, useRef } from 'react';
+import styled from '../../src/styled-components';
+
+const Button = styled.button`
+  background-color: #fff;
+  color: #333;
+  padding: 2px 10px;
+  touch-action: manipulation;
+  cursor: pointer;
+  user-select: none;
+  border: 1px solid #ccc;
+  font-size: 16px;
+  height: 28px;
+  box-sizing: border-box;
+  vertical-align: middle;
+  line-height: 1;
+  outline: none;
+  white-space: nowrap;
+  @media (max-width: 699px) {
+    display: none;
+  }
+`;
+
+function GetUseRef(): RefObject<HTMLInputElement> {
+  return useRef<HTMLInputElement>(null);
+}
+
+const FileInput = props => {
+  const hiddenFileInput: RefObject<HTMLInputElement> = GetUseRef();
```

Comment:
we can use it implicit `useRef<HTMLInputElement>(null)`, wrap func `GetUseRef` not needed